pkg/consensus/blocks_validation.go

```diff
@@ -88,11 +88,11 @@ func (cv *Validator) posAlgo(height uint64) (PosCalculator, error) {
 			return nil, err
 		}
 		if blockV5 {
-			return &FairPosCalculatorV2{}, nil
+			return NewFairPosCalculator(cv.settings.DelayDelta, cv.settings.MinBlockTime), nil
```

Comment:
I can't find initialization of these fields. What if `cv.settings.DelayDelta` or `cv.settings.MinBlockTime` will be uninitialized?